.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@Rendxn
```

Comment:
Add EOF. You can setup VSCode to do it for you: https://stackoverflow.com/a/44704969